CesiumGltf/include/CesiumGltf/PropertyArrayView.h

```diff
@@ -69,7 +81,7 @@ template <> class PropertyArrayView<bool> {
   /**
    * @brief Constructs an empty array view.
    */
-  PropertyArrayView() : _values{}, _bitOffset{0}, _size{0} {}
+  PropertyArrayView() : _values{} {}
```

Comment:
I changed this around and didn't properly revert it, so ignore the missing initialization / random `= 0`'s in the variable declaration